packages/tools/src/utilities/calibratedUnits.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Extracts the length units and the type of calibration for those units
+ * into the response.  The length units will typically be either mm or px
+ * while the calibration type can be any of a number of different calibraiton types.
+ *
+ * Volumetric images have no calibration type, so are just the raw mm.
+ *
+ * TODO: Handle region calibration
+ *
+ * @param handles - used to detect if the spacing information is different
+ *   between various points (eg angled ERMF or US Region)
+ * @param image - to extract the calibration from
+ * @param image.calibration - calibration value to extract units form
+ * @returns String containing the units and type of calibration
+ */
+const calibratedLengthUnits = (handles, image): string => {
```

Comment:
`calibratredLengthUnits` does not give me impression of this is a function rather it is a variable. How about `getCalibratedLengthUnits` and same for area.